stylus/cozy-ui/defaults.styl

```diff
@@ -102,6 +102,9 @@ body
     &.coz-btn-alert--success
         @extend $button-alert--success
 
+:global .coz-btn--upload-menu-item
```

Comment:
Yes, the point is it does not fit with other button classes `coz-btn` and `coz-btn--upload`, as it is contained in a menu. See https://marvelapp.com/139hdh2/screen/17497373